src/interface/update.cpp

```diff
@@ -64,45 +68,98 @@ TaskStatus FluxDivergence(std::shared_ptr<Container<Real>> &in,
   return TaskStatus::complete;
 }
 
-void UpdateContainer(std::shared_ptr<Container<Real>> &in,
-                     std::shared_ptr<Container<Real>> &dudt_cont, const Real dt,
-                     std::shared_ptr<Container<Real>> &out) {
+TaskStatus FluxDivergenceMesh(std::shared_ptr<MeshData<Real>> &in_obj,
```

Comment:
I really dislike having there be a separate `FluxDivergeneceMesh` and `FluxDivergenceBlock` is there not some way these can be the same function, just templated?

Reply:
I generally agree. 
As @jdolence mentioned this is likely more involved than it looks at first sight.
I could imagine a cleaner solution with updated wrappers (so that the wrappers now whether is Block or Mesh data and handling the accesses transparently) or trying to leverage the variable indexing of the `ParArrayNDs` to make this happen (though I'm not sure/didn't check if all the `ParArrayNDs` features survive the packing in general).
Here, I also vote for putting this on the list of things to fix in a separate PR later.

Reply:
I'm willing to fix it in a later PR, sure. So long as we make an issue to remind us to do it.

Reply:
Keeping track in #349